### PR TITLE
Use trusted certificates in RHTAP

### PR DESCRIPTION
**Pull request by flacatus**

### CODE REVIEW

File: ci-operator/step-registry/redhat-appstudio/rhtap-installer/install-rhtap/redhat-appstudio-rhtap-installer-install-rhtap-commands.sh [hunk 1]
```diff
@@ -46,9 +50,27 @@ TPA__OIDC__TESTING_USER_CLIENT_SECRET="0e6bf990-43b4-4efb-95d7-b24f2b94a525" # n
...
+TAS__SECURESIGN__FULCIO__OIDC__CLIENT_ID="fake-one"
+TAS__SECURESIGN__FULCIO__OIDC__TYPE="dex"
+NAMESPACE=rhtap
 
+yq -i 'del(.clusters[].cluster.certificate-authority-data) | .clusters[].cluster.insecure-skip-tls-verify=true' $KUBECONFIG
+OPENSHIFT_PASSWORD="$(cat $KUBEADMIN_PASSWORD_FILE)"
 
-NAMESPACE=rhtap
+timeout --foreground 5m bash  <<- "EOF"
+    while ! oc login "$OPENSHIFT_API" -u kubeadmin -p "$OPENSHIFT_PASSWORD" --insecure-skip-tls-verify=true; do
```
Comment (xinredhat) [inline ci-operator/step-registry/redhat-appstudio/rhtap-installer/install-rhtap/redhat-appstudio-rhtap-installer-install-rhtap-commands.sh:65]: why do you need to login to the openshift? it already exported the KUBECONFIG by default.

Comment (flacatus) [reply ci-operator/step-registry/redhat-appstudio/rhtap-installer/install-rhtap/redhat-appstudio-rhtap-installer-install-rhtap-commands.sh:65]: It is exported with old certificates and will not connect after adding the new ones

Comment (xinredhat) [reply ci-operator/step-registry/redhat-appstudio/rhtap-installer/install-rhtap/redhat-appstudio-rhtap-installer-install-rhtap-commands.sh:65]: get it. htanks

File: ci-operator/step-registry/redhat-appstudio/rhtap-installer/install-rhtap/redhat-appstudio-rhtap-installer-install-rhtap-commands.sh [hunk 2]
```diff
@@ -23,7 +23,11 @@ export ACS__API_TOKEN \
   TPA__POSTGRES__TPA_PASSWORD \
   SPRAYPROXY_SERVER_URL \
   SPRAYPROXY_SERVER_TOKEN \
-  DEVELOPER_HUB__QUAY_TOKEN__ASK_THE_INSTALLER_DEV_TEAM
+  DEVELOPER_HUB__QUAY_TOKEN__ASK_THE_INSTALLER_DEV_TEAM \
+  OPENSHIFT_API \
+  OPENSHIFT_PASSWORD \
+  TAS__SECURESIGN__FULCIO__ORG_EMAIL \
+  TAS__SECURESIGN__FULCIO__ORG_NAME
```
Comment (xinredhat) [inline ci-operator/step-registry/redhat-appstudio/rhtap-installer/install-rhtap/redhat-appstudio-rhtap-installer-install-rhtap-commands.sh:31]: you need to export the following variables
```
  TAS__SECURESIGN__FULCIO__OIDC__URL \
  TAS__SECURESIGN__FULCIO__OIDC__CLIENT_ID \
  TAS__SECURESIGN__FULCIO__OIDC__TYPE
```

**Comment by xinredhat** [comment]
you also need to remove the following line , TPA should work now
```
yq e -i '.trusted-profile-analyzer=null' private-values.yaml
```

**Pull request merged**